config.yaml

```diff
@@ -29,7 +29,8 @@ developer.mode.enabled: false
 # Authentication configuration
 # -----------------------------
 
-token.privatekey : `-----BEGIN RSA PRIVATE KEY-----
+token.privatekey : > 
```

Comment:
If I understand [this document](http://symfony.com/doc/current/components/yaml/yaml_format.html) correctly, the consecutive lines need to be prefix with spaces, no?